AElf.Contracts.Consensus.DPoS/DPoSContractState.cs

```diff
@@ -0,0 +1,185 @@
+using System;
+using AElf.Common;
+using AElf.Kernel;
+using AElf.Sdk.CSharp.State;
+using Google.Protobuf.WellKnownTypes;
+
+namespace AElf.Contracts.Consensus.DPoS
+{
+    public class RoundsMapState : MappedState<UInt64Value, Round>
+    {
+    }
+
+    public class MinersMapState : MappedState<UInt64Value, Miners>
+    {
+    }
+
+    public class TicketsMapState : MappedState<StringValue, Tickets>
+    {
+    }
+
+    public class SnapshotMapState : MappedState<UInt64Value, TermSnapshot>
+    {
+    }
+
+    public class AliasesMapState : MappedState<StringValue, StringValue>
+    {
+    }
+
+    public class AliasesLookupMapState : MappedState<StringValue, StringValue>
+    {
+    }
+
+    public class HistoryMapState : MappedState<StringValue, CandidateInHistory>
+    {
+    }
+
+    public class AgeToRoundNumberMapState : MappedState<UInt64Value, UInt64Value>
+    {
+    }
+
+    public class VotingRecordsMapState : MappedState<Hash, VotingRecord>
+    {
+    }
+
+    public class TermToFirstRoundMapState : MappedState<UInt64Value, UInt64Value>
+    {
+    }
+
+    // ReSharper disable InconsistentNaming
+    public class DividendContractReferenceState : ContractReferenceState
+    {
+        public Action<ulong> KeepWeights { get; set; }
```

Comment:
Action<**> ?

Reply:
Action is used to send inline call. The actual delegate is injected.